src/components/vanilla/charts/KPIChart/index.tsx

```diff
@@ -31,7 +32,7 @@ export default (props: Props) => {
     const prev = parseFloat(prevResults?.data?.[0]?.[metric.name] || 0);
 
     return {
-      percentage: prev ? Math.round((n / prev) * 100) - 100 : null,
+      percentage: prev !== null && prev !== undefined ? Math.round((n / prev) * 100) - 100 : null,
```

Comment:
prev !== null && prev !== undefined 


is equal to **prev** , is this because of the 0 ?

Reply:
@mad-raccoon yes, I want to it calculate infinity for values of 0, as that it then shown below.

Reply:
@mad-raccoon you ok for me to resolve and merge?

Reply:
yes, sorry for the delay.  this has more readability - but i am ok with that as well:

```suggestion
      percentage: prev || prev === 0 ? Math.round((n / prev) * 100) - 100 : null,
```